src/components/LobbyPlayerIndicator.vue

```diff
@@ -33,6 +33,20 @@ export default {
       default: false,
     },
   },
+  watch: {
+    playerUsername: function(newVal, oldVal) {
+      // Play audio when player join lobby
+      if (oldVal === null && newVal !== null) {
+        const audio = new Audio('/public/sounds/lobby/enter-lobby.mp3');
+        audio.play();
+      }
+      // Play audio when player leave lobby
+      else if (oldVal !== null && newVal === null) {
+        const audio = new Audio('/public/sounds/lobby/leave-lobby.mp3');
+        audio.play();
+      }
```

Comment:
I don't think we should be directly adding sounds to the markup as audio tags since those will be get parsed and requested as part of the waterfall. See this example I put together in StackBlitz showing two methods of import (also added some notes in the comments). https://stackblitz.com/edit/vue-u5ehus?file=src%2FApp.vue,src%2Fcomponents%2FPlaySound.vue,src%2Faudio%2Fenter-lobby.js

cc @itsalaidbacklife